components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -559,7 +588,27 @@ interface ReadableHistoryConnection : InterruptibleConnection {
      * @param start The (inclusive) start time to bound the query.
      * @param end The (inclusive) end time to bound the query.
      */
-    fun getVisitInfos(start: Long, end: Long = Long.MAX_VALUE): List<VisitInfo>
+    fun getVisitInfos(
+        start: Long,
+        end: Long = Long.MAX_VALUE,
+        excludeTypes: List<VisitType> = listOf()
+    ): List<VisitInfo>
+
+    /**
+     * Return a "page" of history results.
```

Comment:
Pagination implies stable ordering. How these results are ordered is currently not specified in the doc. Can you add that? I expect that this is in descending order by timestamp.